operators/pkg/controller/license/license_controller.go

```diff
@@ -55,14 +55,7 @@ func (r *ReconcileLicenses) Reconcile(request reconcile.Request) (reconcile.Resu
 	defer func() {
 		log.Info("End reconcile iteration", "iteration", currentIteration, "took", time.Since(iterationStartTime), "namespace", request.Namespace, "es_name", request.Name)
 	}()
-	result, err := r.reconcileInternal(request)
-	if result.Requeue {
-		log.Info("Re-queuing new license check immediately (rate-limited)", "namespace", request.Namespace, "es_name", request.Name)
-	}
-	if result.RequeueAfter > 0 {
-		log.Info("Re-queuing new license check", "namespace", request.Namespace, "es_name", request.Name, "RequeueAfter", result.RequeueAfter)
-	}
-	return result, err
```

Comment:
This does not seem to bring much value (looks like we are debugging the requeue mechanism)?